doc/release-notes/6426-reexport-all

```diff
@@ -0,0 +1,14 @@
+## Notes for Dataverse Installation Administrators
+
+### Run ReExportall
+
+We made changes to the JSON Export in this release (#6246). If you'd like these changes to reflected in your JSON exports, you should run ReExportall as part of the upgrade process. We've included this in the detailed instructions below.
```

Comment:
"We made changes to the JSON Export" sounds a little scary to me but we can reword this when we write the actual release note. What we're really doing is populating a data field with a date instead of the word "Production Date".